src/uis/uiKick.tsx

```diff
@@ -182,6 +205,108 @@ export class KickUI {
     )
   }
 
+  createUnKickUi(): ReactEcs.JSX.Element | null {
+    if (this.gameController.uiController.canvasInfo === null) return null
+    return (
+      <UiEntity
+        uiTransform={{
+          flexDirection: 'column',
+          width: this.gameController.uiController.canvasInfo.width,
+          height: this.gameController.uiController.canvasInfo.height,
+          justifyContent: 'center',
+          alignItems: 'center',
+          display: this.unKickUiVisibility ? 'flex' : 'none'
+        }}
+      >
+        <UiEntity
+          uiTransform={{
+            flexDirection: 'column',
+            alignItems: 'center',
+            justifyContent: 'center',
+            positionType: 'absolute',
+            width: 550 * getScaleFactor(), // Use scale factor to make it responsive
+            height: 300 * getScaleFactor(), // Use scale factor to make it responsive
+            borderRadius: 15
+          }}
+          uiBackground={{ color: Color4.White() }}
+        >
+          <Label
+            value="Select Player to Unban"
+            fontSize={24 * getScaleFactor()}
+            color={Color4.Black()}
+            uiTransform={{
+              width: 300 * getScaleFactor(),
+              height: 60 * getScaleFactor(),
+              alignContent: 'center',
+              margin: '20px 20px 20px 20px'
+            }}
+          />
+          <UiEntity
+            uiTransform={{
+              flexDirection: 'row', // Set to 'row' to align children (Input, Label) side by side
+              width: 400 * getScaleFactor(),
+              height: 50 * getScaleFactor(),
+              alignItems: 'center',
+              justifyContent: 'center'
+            }}
+          >
+            <Dropdown
+              options={[
+                'Select Player',
+                ...this.gameController.playersOnScene.getTaggedNamesFromWallets(
+                  BannedComponent.get(this.bannedEntity).list
+                )
+              ]}
```

Comment:
Since we don't want `Select Player` to be a selectable option, I think it'd be better to use the `emptyLabel` field for this:

```suggestion
            <Dropdown
              emptyLabel='Select Player'
              options={this.gameController.playersOnScene.getTaggedNamesFromWallets(
                  BannedComponent.get(this.bannedEntity).list
                )
              }
```